### PR TITLE
Bump jörmungandr to 0.8.7

### PR DESCRIPTION
**Pull request by Anviking**
- [x] Ge-generated block0
- [ ] <s>Fully updated hashes in jormungandr nix file</s>
- [x] ⚠️ Use jormungandr derivation from iohk-nix (drastic change) https://input-output-rnd.slack.com/archives/CAP8NM7N0/p1579865455063700?thread_ts=1579861205.062100&cid=CAP8NM7N0
- [x] Updated readme

### CODE REVIEW

**Comment by Anviking** [comment]
bors try

**Comment by Anviking** [comment]
This actually seems to almost work 🤨

**Review by paweljakubas — APPROVED** [review]
LGTM, one more thing please update matrix - master -> 0.8.7

**Comment by Anviking** [comment]
The changes to `nix/jormungandr.nix` were kind of drastic. I wonder if e.g windows builds are working properly. Not sure what is enforced and where. The non-required CI checks are failing.

Or these failures are expected?

File: nix/jormungandr.nix [hunk 1]
```diff
@@ -34,38 +34,8 @@
...
-  jormungandr-cli = nonWindows (iohkLib.rust-packages.pkgs.makeJcli release);
-
-  inherit jormungandr-win64;
-  inherit (jormungandr) src;
-}
+  iohk-nix = import (fetchTarball {
+    url = https://github.com/input-output-hk/iohk-nix/archive/e0b995585478f1e2fb417123cc224054092090a1.tar.gz;
+    sha256 = "0igigjk54y8d50wpwbx3p9biqa098af95xjmi50sy6f5sszjvk3s";
+  }) {};
+in { inherit (iohk-nix.jormungandrLib.packages.v0_8_7) jormungandr jcli; }
```
Comment (KtorZ) [inline nix/jormungandr.nix]: Hmmm. This seems wrong. The above was made in order to enable us to use a different version that the one used by iohk-nix. Often, iohk-nix is lagging behind or, on the contrary, sometimes ahead, breaking our own CI. We are managing the whole version of Jörmungandr ourselves on purpose.

What was the reason to remove all this?

Comment (Anviking) [reply nix/jormungandr.nix]: Reason: circumvent the build failure I was seeing. https://input-output-rnd.slack.com/archives/CAP8NM7N0/p1579861205062100

Alternatively, we could bump the rust version (as Michael suggested, and seems logical).

But I am not sure where the rust version is specified or how difficult doing that would be 🤔

**Comment by piotr-iohk** [comment]
bors try

File: nix/jormungandr.nix (hunk 1 above)
Comment (piotr-iohk) [reply nix/jormungandr.nix]: I suppose that's the way to go -> https://github.com/input-output-hk/cardano-wallet/pull/1304/commits/900f3f7477ba6644b718f7c5e4dfbd1d0a381a45

After this and bumping all hashes to jörmungandr 0.8.7 I was able to run successfully on local box:

```
nix-build nix/jormungandr.nix
```

**Comment by Anviking** [comment]
Unrelated failure, will have a look (<s>looks like overflow</s>), bors r+

**Comment by piotr-iohk** [comment]
Failed:
```

Failures:
--
  |  
  | test/unit/Cardano/Wallet/Jormungandr/RewardsSpec.hs:31:5:
  | 1) Cardano.Wallet.Jormungandr.Rewards.rewardsAt try-out ITN parameters
  | Falsifiable (after 53 tests):
  | EpochNo {unEpochNo = 0}
  | Quantity {getQuantity = 0} /= Quantity {getQuantity = 3452054796000}
  |  
  | To rerun use: --match "/Cardano.Wallet.Jormungandr.Rewards/rewardsAt/try-out ITN parameters/"
  |  
```

Updated -> https://github.com/input-output-hk/cardano-wallet/issues/1305

**Pull request merged**